packages/forms/src/templates/FieldTemplate.js

```diff
@@ -38,7 +39,7 @@ const CustomFieldTemplate = ({
 			<Label className="form-label" label={label} required={required} id={id} />}
 			{children}
 			{!hasLabelBefore && !isToggle && displayLabel &&
-			<Label label={label} required={required} id={id} className="control-label" />}
+			<Label label={label} required={required || schema.required} id={id} className="control-label" />}
```

Comment:
Discussed with @jare-talend, the array notation is used in schema-form v4. There's an issue on required field validation, this PR is a workaround for this issue but the required in property form was used in v3.
We should wait for the validation issue to be fixed in v4, so that we don't make unnecessary change in backend.